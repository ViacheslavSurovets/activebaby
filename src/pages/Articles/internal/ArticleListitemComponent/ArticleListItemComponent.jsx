import React from 'react';
import { ArticleListItem, ArticleListItemContainer } from './styles';
import PropTypes from 'prop-types';
import { useTranslation } from 'react-i18next';

const ArticleListItemComponent = ( { article } ) => {
  const { t } = useTranslation ();
  return (
    <ArticleListItemContainer>
      <ArticleListItem to={ article.routePath }>{ t ( article.title ) }</ArticleListItem>
    </ArticleListItemContainer>
  );
};

export default ArticleListItemComponent;

ArticleListItemComponent.propTypes = {
  article: PropTypes.object
};

