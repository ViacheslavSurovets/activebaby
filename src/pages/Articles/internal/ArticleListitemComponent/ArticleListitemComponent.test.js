import React from 'react';
import { shallow } from 'enzyme';
import ArticleListItemComponent from './ArticleListItemComponent';


describe ( 'ArticleListItemComponent', () => {
  const mockArticle = {
    routePath: 'string',
    title: 'string'
  };
  it ( 'ArticleListItemComponent Snapshot', () => {
    expect ( shallow ( <ArticleListItemComponent article={ mockArticle } /> ) ).toMatchSnapshot ();
  } );
} );
