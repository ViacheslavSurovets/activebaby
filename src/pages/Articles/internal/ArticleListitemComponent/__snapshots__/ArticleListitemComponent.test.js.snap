// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleListItemComponent ArticleListItemComponent Snapshot 1`] = `ShallowWrapper {}`;
